AmountInput/AmountInput.hpp

```diff
@@ -0,0 +1,8 @@
+namespace tvm { namespace schema {
+
+__interface IAmountInput {
+	[[internal, answer_id]]
+	uint256 get(string prompt, uint8 decimals, uint256 min, uint256 max);
+}
```

Comment:
add function without interval
uint128 get(string prompt, uint8 decimals);
uint128 getFromInterval(string prompt, uint8 decimals, uint128 min, uint128 max);

not bad idea to add function for tons input, but i'm not sure..
uint128 getTons(string prompt);
uint128 getTonsFromInterval(string prompt, uint128 min, uint128 max);

Reply:
We decided to use current `get` method. Because it allows to work with any currency and enables additional verification on debot browser side.